plugins/modules/ovirt_disk.py

```diff
@@ -765,7 +765,7 @@ def searchable_attributes(module):
     attributes = {
         'name': module.params.get('name'),
         'Storage.name': module.params.get('storage_domain'),
-        'vm_names': module.params.get('vm_name'),
+        'vm_names': module.params.get('vm_name') if module.params.get('state') != 'attached' else None,
```

Comment:
Now after looking at it again, have 1 question, should this `vm_name` filtering by `state` should be be done here on when used according to the flow?
I mean this `searchable_attributes()` looks like a pretty generic method, that has a potential to be used in multiple places.

Reply:
If it's worth, please do that in followup patch